src/domain/templates/components/Dialogs/ApplyCollaborationTemplateDialog/ApplyCollaborationTemplateDialog.tsx

```
@@ -0,0 +1,98 @@
+import { Dialog, DialogActions, Button, DialogContent, RadioGroup, FormControlLabel, Radio } from '@mui/material';
+import DialogHeader from '../../../../../core/ui/dialog/DialogHeader';
+import { Trans, useTranslation } from 'react-i18next';
+import { Caption, Text } from '../../../../../core/ui/typography';
+import useLoadingState from '../../../../shared/utils/useLoadingState';
+import { LoadingButton } from '@mui/lab';
+import Gutters from '../../../../../core/ui/grid/Gutters';
+
+interface ApplyCollaborationTemplateDialogProps {
+  open: boolean;
+  onClose: () => void;
+  onConfirm: (addCallouts: boolean) => Promise<unknown> | void;
```

Biome

[error] void is confusing inside a union type.

Unsafe fix: Use undefined instead.


(lint/suspicious/noConfusingVoidType)